src/stable_browser.ts

```
@@ -810,7 +830,66 @@
     }
     return result;
   }
-
+  async simpleClick(elementDescription, _params?: Params, options = {}, world = null) {
+    const startTime = Date.now();
+    let timeout = 30000;
+    if (options && options.timeout) {
```

Biome

[error] Change to an optional chain.

Unsafe fix: Change to an optional chain.


(lint/complexity/useOptionalChain)